index.js

```diff
@@ -2,18 +2,15 @@
 
 require('dotenv').config({silent: true});
 
-const myFTClient = require('./lib/myFTClient');
-const acquisitionCtxClient = require('./lib/acquisitionCtxClient');
-const userProfileClient = require('./lib/userProfileClient');
-const clientErrors = require('./lib/statusErrors');
-const accessLicenceClient = require('./lib/accessLicenceClient');
-const kinesisClient = require('./lib/kinesisClient');
+const fs = require('fs');
+const paths = [
+	'./lib'
+];
```

Comment:
What is the likelihood of us having multiple paths? Just thinking we could simplify it and eliminate the need for `paths.forEach`.